sdk/php/runtime/template/README.md

```diff
@@ -311,3 +311,63 @@ class Example
      // ...
 }
 ```
+
+#### Directories and Files
+
+[Directories and Files can also specify additional meta data.](https://docs.dagger.io/manuals/developer/functions/#directories-and-files)
+
+
+##### Default Values
+
+It would be tricky to instantiate a `Directory` or `File` object from scratch.
+So the standard way of specifying a default value in PHP is hard to apply.
+
+Instead specify a `DefaultPath` attribute with a `string` path.
+Dagger will do the heavy lifting of creating the default object for you.
+
+Paths are relative to the `source` provided in the `dagger.json` file.
```

Comment:
I still do not understand this part.

Here are the 3 cases:
```
If an absolute path is specified:
- in a Git repository (defined by the presence of a .git sub-directory), the default context is the root of the Git repository.
- in a non-repository location (defined by the absence of a .git sub-directory), the default context is the directory containing a dagger.json file.

If a relative path is specified the default context is always the directory containing a dagger.json file.
```

Reply:
Apologies, @helderco solved my confusion.

I was testing default paths on an outdated version of Dagger where it was relative to `source`. I've fixed this now and literally copy-pasted your description.

Thank you, for patiently reviewing this :)

Reply:
I've updated it once more

https://github.com/dagger/dagger/blob/d925956f33d1dfcdbb3b0e163b2bcf3d053eecc8/sdk/php/runtime/template/README.md?plain=1#L328-L359